src/legacy/core_plugins/console/public/src/components/settings_modal.tsx

```diff
@@ -0,0 +1,263 @@
+      fontSize,
+      wrapMode,
+      autocomplete: {
+        fields,
+        indices,
+        templates,
+      },
+      polling,
+      tripleQuotes,
+    });
+
+    setIsLoading(false);
+  }
+
+  return (
+    <EuiOverlayMask>
+      <EuiModal
+        data-test-subj="devToolsSettingsModal"
+        className="devToolsSettingsModal"
+        onClose={props.onClose}
+      >
+        <EuiModalHeader>
+          <EuiModalHeaderTitle>
+            <FormattedMessage id="console.settingsPage.pageTitle" defaultMessage="Settings" />
+          </EuiModalHeaderTitle>
+        </EuiModalHeader>
+
+        <EuiModalBody className="euiText euiText--medium euiText--constrainedWidth">
+          <EuiFlexGrid columns={1}>
+            <EuiFlexItem>
+              <EuiFlexItem>
+                <EuiText>
+                  <p>
+                    <FormattedMessage
+                      id="console.settingsPage.fontSizeLabel"
+                      defaultMessage="Font Size"
+                    />
+                  </p>
+                </EuiText>
+              </EuiFlexItem>
+              <EuiFlexItem>
+                <EuiFieldNumber
+                  autoFocus
+                  data-test-subj="fontSizeInput"
+                  value={fontSize}
+                  onChange={e => {
+                    setFontSize((e.target.value as unknown) as number);
+                  }}
+                />
+              </EuiFlexItem>
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiSwitch
+                checked={wrapMode}
+                data-test-subj="settingsWrapLines"
+                id="wrapLines"
+                label={
+                  <FormattedMessage
+                    defaultMessage="Wrap Long Lines"
+                    id="console.settingsPage.wrapLongLinesLabelText"
+                  />
+                }
+                onChange={e => setWrapMode(e.target.checked)}
+              />
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <h4>
+                <FormattedMessage
+                  id="console.settingsPage.jsonSyntaxLabel"
+                  defaultMessage="JSON syntax"
+                />
+              </h4>
+              <EuiSwitch
+                checked={tripleQuotes}
+                data-test-subj="tripleQuotes"
+                id="tripleQuotes"
+                label={
+                  <FormattedMessage
+                    defaultMessage="Use tripple quotes in output pane"
+                    id="console.settingsPage.tripleQuotesMessage"
+                  />
+                }
+                onChange={e => setTripleQuotes(e.target.checked)}
+              />
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <h4>
+                <FormattedMessage
+                  id="console.settingsPage.autocompleteLabel"
+                  defaultMessage="Autocomplete"
+                />
+              </h4>
+              <EuiSwitch
```

Comment:
These should actually be changed back to checkboxes and wrapped in a legend style form row like:

```jsx
<EuiFormRow
  label="Autocomplete"
  labelType="legend">
          <EuiCheckboxGroup
            options={this.state.checkboxes}
            idToSelectedMap={this.state.checkboxIdToSelectedMap}
            onChange={this.onCheckboxChange}
          />
</EuiFormRow>
```